charts/palisade-service/templates/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: {{ include "palisade-service.name" . }}
+    namespace: {{ include "palisade.namespace" . }}
+    labels:
+        app.kubernetes.io/name: {{ include "palisade-service.name" . }}
+        helm.sh/chart: {{ include "palisade-service.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+    replicas: {{ .Values.replicaCount }}
+    selector:
+        matchLabels:
+            app.kubernetes.io/name: {{ include "palisade-service.name" . }}
+            app.kubernetes.io/instance: {{ .Release.Name }}
+    template:
+        metadata:
+            labels:
+                app.kubernetes.io/name: {{ include "palisade-service.name" . }}
+                app.kubernetes.io/instance: {{ .Release.Name }}
+        spec:
+            containers:
+                - name: {{ .Chart.Name }}
+                  image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+                  imagePullPolicy: {{ .Values.image.pullPolicy }}
+                  ports:
+                      - name: web
+                        containerPort: 8084
+                        protocol: TCP
+                  livenessProbe:
+                      httpGet:
+                          path: /actuator/health
+                          port: 8084
+                      initialDelaySeconds: 10
+                      failureThreshold: 5
+                      periodSeconds: 20
+                      timeoutSeconds: 5
+                  readinessProbe:
+                      httpGet:
+                          path: /actuator/health
+                          port: 8084
```

Comment:
Palisade service used to be on 8080, is there a need to move it to 8084?

Reply:
Pods are exposed to the cluster via the service on port 80 so the internal port is immaterial. Also convention places dashboards on 8080, as is the traefik dashboard (external ip) so wanted to avoid confusion.